### PR TITLE
test: add test case for op stack e2e test

### PR DESCRIPTION
**Pull request by lesterli**
## Summary

This PR adds a test case for the op stack e2e test.

## Test Plan

```
make test
make test-e2e
```

### CODE REVIEW

**Review by SebastianElvis — APPROVED** [review]

File: itest/bytecode/op_finality_gadget_1947cc6.wasm
Comment (maurolacy) [inline itest/bytecode/op_finality_gadget_1947cc6.wasm:1]: Nit: Use a fixed name, and a `version.txt` file to indicate the version / commit.

File: itest/opstackl2/op_test_manager.go [hunk 1]
```diff
@@ -36,12 +38,10 @@ import (
 )
 
 const (
-	opFinalityGadgetContractPath = "../bytecode/op_finality_gadget_c6ccca8.wasm"
+	opFinalityGadgetContractPath = "../bytecode/op_finality_gadget_1947cc6.wasm"
```
Comment (maurolacy) [inline itest/opstackl2/op_test_manager.go:41]: Then you can avoid this.

File: itest/opstackl2/op_test_manager.go [hunk 2]
```diff
@@ -167,8 +168,10 @@ func mockOpL2ConsumerCtrlConfig(nodeDataDir string) *fpcfg.OPStackL2Config {
 
 	// fill up the config from dc config
 	return &fpcfg.OPStackL2Config{
-		OPStackL2RPCAddress:     opStackL2RPCAddress,
-		OPFinalityGadgetAddress: opFinalityGadgetAddress,
+		OPStackL2RPCAddress: opStackL2RPCAddress,
+		// make random contract address for now to avoid validation errors,
+		// later we will update it with the correct address in the test
```
Comment (maurolacy) [inline itest/opstackl2/op_test_manager.go:173]: Nit: use `TODO:`.

Comment (bap2pecs) [reply itest/opstackl2/op_test_manager.go:173]: yes it's already fixed. latest code

```
opcc.Cfg.OPFinalityGadgetAddress = resp.Contracts[0]
```

Comment (bap2pecs) [reply itest/opstackl2/op_test_manager.go:173]: this is probably not relevant. let me check the latest code to see how to fix it

Comment (bap2pecs) [reply itest/opstackl2/op_test_manager.go:173]: addressed in https://github.com/babylonchain/finality-provider/pull/483

**Pull request merged**